code/game/objects/items/weapons/twohanded.dm

```diff
@@ -333,6 +334,9 @@
 
 	sleep(3)
 
+	if(gib_user)
+		user.gib()
+
 	var/turf/epicenter = get_turf(target)
 	target.ex_act(400, null, src, user, 100)
 	cell_explosion(epicenter, detonation_force, 50, EXPLOSION_FALLOFF_SHAPE_LINEAR, null, create_cause_data(initial(name), user))
```

Comment:
```suggestion

	addtimer(CALLBACK(src, PROC_REF(lungemine_detonate), target, user), 0.3 SECONDS)

/obj/item/weapon/twohanded/lungemine/proc/lungemine_detonate(atom/target, mob/user)
	var/turf/epicenter = get_turf(target)
	cell_explosion(epicenter, detonation_force, 50, EXPLOSION_FALLOFF_SHAPE_LINEAR, null, create_cause_data(initial(name), user))
	if(gib_user)
		user.gib()
```

Reply:
- use addtimer rather than sleep
- the ex_act proc was formatted incorrectly and didn't actually apply to xenos at all, it could be fixed by removing `user` from the args, however this makes the lungemine gib almost anything you hit with it (even crushers) which would be a large departure from the current behavior (e.g., people are used to it being bugged to ignore the ex_act anyway.) Removing it has the side effect of making it less effective against preds.
either remove the `user` arg from the ex_act or remove the line entirely as in this suggestion.
Also moved the gib to the end, this is because `cell_explosion` references the `user`, therefore we should wait to destroy the mob that is `user` until after that code has run.